app/classifier/index.cjsx

```diff
@@ -184,23 +182,6 @@ Classifier = React.createClass
             annotation={currentAnnotation}
             subjectLoading={@state.subjectLoading}
           >
-            <TaskNav
-              annotation={currentAnnotation}
-              classification={currentClassification}
-              completeClassification={@completeClassification}
-              project={@props.project}
-              subject={@props.subject}
-              task={currentTask}
-              workflow={@props.workflow}
-            >
-              {if @props.expertClassifier
-                <ExpertOptions
-                  classification={currentClassification}
-                  userRoles={@props.userRoles}
-                  demoMode={@props.demoMode}
-                  onChangeDemoMode={@props.onChangeDemoMode}
-                />}
-            </TaskNav>
             <p>
```

Comment:
Tutorial buttons etc. here should be rendered after the task navigation buttons.